src/backend/aspen/app/app.py

```diff
@@ -180,7 +180,7 @@ def decorated(*args, **kwargs):
             return redirect("/login")
         with session_scope(application.DATABASE_INTERFACE) as db_session:
             g.db_session = db_session
-            setup_userinfo(auth0_user_id)
+            g.auth_user = setup_userinfo(auth0_user_id)
```

Comment:
I think this is a little clearer what's going on (imo, totally cool if you think this is not a better idea lol)
```suggestion
            found_auth_user = setup_userinfo(auth0_user_id)
            if not found_auth_user:
                # login attempt from user not in DB
                # TODO: return response to user that they do not have an account with aspen.
                return redirect("/login")
            else:
                g.auth_user = found_auth_user
            
```

Reply:
merged!